src/commands/install.yml

```diff
@@ -6,7 +6,7 @@ parameters:
     type: string
     default: latest
   python_version:
-    description: 'If set, this version of Python will be installed and set with pyenv globally. ex: "3.7.0" This is only for the local environment and will not have any effect if use-container is enabled.'
+    description: 'If set, this version of Python will be installed and set with pyenv globally. ex: "3.7.0" This is only for the local environment and will not have any effect if use_container is enabled.'
```

Comment:
Is `use-container` an orb-related thing? If not, then it should remain unchanged.

Reply:
That's talking about the actual `use_container` parameter. It's a boolean value that defaults to `true`